### PR TITLE
db config

### PR DESCRIPTION
**Pull request by WarrenMfg**

### CODE REVIEW

File: database/config.js [hunk 1]
```diff
@@ -1,16 +1,149 @@
...
+};
+
+seed();
+
+module.exports = seed;
+//
+
+
+
+// AMAZON
```
Comment (JohnCharamonde) [inline database/config.js:142]: what does this Amazon reference?

Comment (WarrenMfg) [reply database/config.js:142]: below that is Jordan's mysql.createConnection settings for AWS. just thought i'd leave it in there for future reference.

File: database/config.js [hunk 2]
```diff
@@ -1,16 +1,149 @@
...
+  let value = faker.finance.amount(0, 5, 1);
 
+  let certOfExcellence = faker.random.number(1);
+  let greenLeaders = faker.random.number(1);
 
+  let aboutCopy = faker.lorem.paragraphs(5, '\n');
 
+  let amenityIcon1 = faker.random.number(10);
+  let amenityCopy1 = faker.lorem.sentence(2);
+  let amenityIcon2 = faker.random.number(10);
```
Comment (JohnCharamonde) [inline database/config.js:20]: Do the copies represent the text that shows on a mouseover?

Comment (WarrenMfg) [reply database/config.js:20]: great question! i plan on addressing the micro-interactions once i get more structure on the frontend. i'll just reseed with added data at that time.

**Review by JohnCharamonde — APPROVED** [review]
Looks really great Kent! Happy to get this code merged in. Couple questions about the database/config-test.js faker code whenever you have a sec.

File: database/config.js (hunk 1 above)
Comment (incogitance) [reply database/config.js:142]: indubitably

File: database/config.js (hunk 2 above)
Comment (incogitance) [reply database/config.js:20]: i ended up asking the same thing... I need to be doing that here though

**Pull request merged**